include/rangers-api/Application/Game/GameModeStage.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+namespace app::game {
+    class GameModeStage : public GameMode {
+    public:
+        class State {
```

Comment:
Starting braces should be started on a line to keep consistent with the original repository.